internal/internal_logging_tags.go

```diff
@@ -36,3 +36,8 @@ const (
 	tagSideEffectID    = "SideEffectID"
 	tagChildWorkflowID = "ChildWorkflowID"
 )
+
+const (
+	tagClientImpl = "ClientImpl" // metrics tag for cadence client implementation
```

Comment:
use
 	// header that contains the client implementation
	clientImplHeaderName  = "cadence-client-name"
	clientImplHeaderValue = "uber-go"

Reply:
these const already defined. do not define new ones.